empyrical/perf_attrib.py

```diff
@@ -0,0 +1,130 @@
+import pandas as pd
+
+
+def perf_attrib(returns,
+                positions,
+                factor_returns,
+                factor_loadings):
+    """
+    Does performance attribution given risk info.
```

Comment:
It would be nice to have a little more information in this docstring about what performance attribution is and what it's for.  Just saying `perf_attrib` is a function that "does performance attribution" doesn't really provide much information to a reader.

Reply:
added a more detailed docstring